qiskit_aer/quantum_info/states/aer_statevector.py

```diff
@@ -35,7 +35,8 @@ class AerStatevector(Statevector):
     def __init__(self, data, dims=None, **configs):
         """
         Args:
-            data (np.array or list or Statevector or QuantumCircuit or qiskit.circuit.Instruction):
+            data (np.array or list or AerStatevector or QuantumCircuit or
```

Comment:
Is `AerStatevector` here correct? We can't pass a `Statevector` object directly to the constructor and build an AerStatevector from it?

Reply:
Currently, this constructor does not take `Statevector` as an initial value. However, it may be better to take it.

```
elif isinstance(data, Statevector)
    data, aer_state = AerStatevector._from_ndarray(np.array(data.data, dtype=complex),
                                                               configs)
```

Reply:
If `Statevector` will be also allowed, we should do

- Add the presented implementation above
- Update documentations
- Add related unittest

Since the aim of this PR is to fix a minor bug in some unittests, it may be better to make a separate PR for implementation improvements.

Reply:
Yeah, we can make this larger change in a new follow up PR after this merges, that makes sense to me